README.md

```diff
@@ -4,4 +4,62 @@ A vue component library and design system for rotki
 
 ## License
 
-[AGPL-3.0](./LICENSE) License &copy; 2023- [Rotki Solutions GmbH](https://github.com/rotki)
\ No newline at end of file
+[AGPL-3.0](./LICENSE) License &copy; 2023- [Rotki Solutions GmbH](https://github.com/rotki)
+
+
+### @rotki/ui-library
+
+To install the dependencies you need to run on the root of the repository
+
+```
+pnpm install
+```
+
+### Compiles and minifies for production
+The following command when executed from the project root will build the `@rotki/ui-library` bundle.
+```
+pnpm run build
+```
+
+### Lint check
+```
+pnpm run lint
+```
+
+### Lints and fixes files
+```
+pnpm run lint:fix
+```
+
+# Storybook
+In order to run the storybook, you can run:
+
+```
+pnpm run storybook
+```
+
+# How to test the usage locally.
+After you build the bundle, in the `package.json` on your main project, you can add this to the dependencies:
+
+```
+"@rotki/ui-library": "file:...path_of_this_directory",
+```
+
+Don't forget to import the `style.css` file from `@rotki/ui-library`
+
+```
+import '@rotki/ui-library/dist/style.css';
+```
+
+And then you can use the component 
+```
+<script setup lang="ts">
+import { Button as RotkiButton } from '@rotki/ui-library';
+</script>
+
+<template>
+  <v-container>
+    <rotki-button label="This is button" outlined />
+  </v-container>
+</template>
+```
```

Comment:
Can we move this above the License? 
The license should be at the bottom of the file.